### ISSUE TITLE
Question - can I disable the tap action 

### ISSUE THREAD

**Issue by direx1** [TarheelGrad1998/gallery-card#49]
Hi, 

Absolutely awesome dashboard card.

Question is, is it possible to disable the tap to fullscreen action. Far too often when I'm clicking next image or using companion app zoom it triggers into full screen. I would be happy to use full screen but it doesn't display correctly as I'm using this card further down in a lovelace view and the image requires me to scroll all the way to the top of the page. Ideally but not essential would be to have a small browser mod style popup but for now any way to edit the module to disable will do.

Thanks.

**Comment by TarheelGrad1998** [comment]
There's nothing built into the code, but yes if you don't mind editing code it's fairly easy.  Edit the gallery-card.js and remove:

1) on line 32, delete this text:
@click="${ev => this._popupCamera(ev)}"

2) on line 38, delete this text:
@click="${ev => this._popupImage(ev)}"

Clear your browser cache after saving.  That should do it.

**Comment by direx1** [comment]
amazing thanks

**Comment by direx1** [comment]
> There's nothing built into the code, but yes if you don't mind editing code it's fairly easy. Edit the gallery-card.js and remove:
> 
> 1. on line 32, delete this text:
>    @click="${ev => this._popupCamera(ev)}"
> 2. on line 38, delete this text:
>    @click="${ev => this._popupImage(ev)}"
> 
> Clear your browser cache after saving. That should do it.

Actually this doesn't work even after clearing cache. I think this is part of the edit panel not the part I'm looking for.

**Comment by TarheelGrad1998** [comment]
No, that's it.  The edit panel is way down at the bottom, >= 866 or so.

Maybe try restarting HA after updating.  I've had to do that before, but I don't understand why.

**Comment by cvjensen** [comment]
hi @TarheelGrad1998 - regarding the popup image , is it possible to swipe/use the buttons when looking at the popup version of the image ?

**Comment by vingerha** [comment]
I tested it in a incognito browser and there it works fine...no tap action on picture anylonger
